### PR TITLE
Manual/pytket 1.20.1

### PR DESCRIPTION
**Pull request by CalMacCQ**
Rebuilt the manual with 

```
pytket == 1.20.1
pytket-qiskit == 0.44.0
pytket-cirq == 0.30.0
```

Points to note
* The manual is now built with an updated dark theme. We've upgraded to pydata_sphinx_theme 0.14.1. This is the cause of the large diff as the html is reformatted
* There is a new deprecation warning with qiskit

```
WARNING: Cell printed to stderr:
/var/folders/p1/hhrjrbd50hq35l9hgk8567tc0000gn/T/ipykernel_7959/1497311364.py:2: DeprecationWarning: ``qiskit.algorithms`` has been migrated to an independent package: https://github.com/qiskit-community/qiskit-algorithms. The ``qiskit.algorithms`` import path is deprecated as of qiskit-terra 0.25.0 and will be removed no earlier than 3 months after the release date. Please run ``pip install qiskit_algorithms`` and use ``import qiskit_algorithms`` instead.
  from qiskit.algorithms import Grover, AmplificationProblem
 ```
 
 It seems we will need the separate `qiskit_algorithms` package in the future.
 
 EDIT: I need to get rid of this warning for the CI checks to pass so I've installed [qiskit-algorithms](https://qiskit.org/ecosystem/algorithms/) it in the `build-manual` script.

### CODE REVIEW

File: docs/manual/_sources/manual_backend.rst [hunk 1]
```diff
@@ -533,7 +533,7 @@ Below we show how the :py:class:`CirqStateSampleBackend` from the ``pytket-cirq`
 .. jupyter-execute::
 
     from qiskit.primitives import BackendSampler
-    from qiskit.algorithms import Grover, AmplificationProblem
+    from qiskit_algorithms import Grover, AmplificationProblem
```
Comment (CalMacCQ) [inline docs/manual/_sources/manual_backend.rst:536]: This is the minor syntax change that is needeed to fix the deprecation warning.

qiskit-algorithms is a separate package

File: manual/manual_backend.rst [hunk 2]
```diff
@@ -533,7 +533,7 @@ Below we show how the :py:class:`CirqStateSampleBackend` from the ``pytket-cirq`
 .. jupyter-execute::
 
     from qiskit.primitives import BackendSampler
-    from qiskit.algorithms import Grover, AmplificationProblem
+    from qiskit_algorithms import Grover, AmplificationProblem
```
Comment (CalMacCQ) [inline manual/manual_backend.rst:536]: This is the change that is needed to fix the deprecation warning


`qiskit-algorithms` is a separate package

File: build-manual [hunk 3]
```diff
@@ -2,6 +2,7 @@
 
 python -m pip install -r manual_requirements.txt
 python -m pip install sphinx sphinx-book-theme sphinx-copybutton jupyter-sphinx
+python -m pip install qiskit-algorithms ~= 0.2.1
```
Comment (cqc-melf) [inline build-manual]: Should this be a requirement of pyket-qiskit in the future?

Comment (cqc-alec) [reply build-manual]: It's a one-line fix to pytket-qiskit that I have on a branch (where I'm trying to fix all the deprecation warnings). I can make a PR with just this fix.

Comment (CalMacCQ) [reply build-manual]: I don't think so.

This is for quantum algorithms so I think it contains things like pre-built circuits for Grover's algorithm etc.

I don't think this is within the scope of pytket-qiskit right now.

Its possible we may need to update the pytket-qiskit tests though as I think `qiskit.algorithms` is used there.

**Review by cqc-melf — DISMISSED** [review]

File: manual_constraints.txt [hunk 4]
```diff
@@ -1,3 +1,4 @@
 sphinx >= 4.5, <6.2.0
 sphinx_autodoc_annotation >= 1.0
-sphinx_book_theme ~= 1.0.1
\ No newline at end of file
+sphinx_book_theme ~= 1.0.1
+qiskit-algorithms ~= 0.2.1
```
Comment (cqc-alec) [inline manual_constraints.txt:4]: I think we should omit this constraint; we can add it if we encounter a problem with 0.3...

**Review by cqc-alec — COMMENTED** [review]
Looks good, just a small comment.

File: manual_constraints.txt (hunk 4 above)
Comment (CalMacCQ) [reply manual_constraints.txt]: Fixed a mistake in the previous commit

**Review by cqc-melf — APPROVED** [review]

**Pull request merged**